bot/src/router.py

```
@@ -252,3 +252,46 @@ async def handle_drop(uid):
+            f"{API_BASE}api/discord/issue_token/",
+            headers={"Authorization": f"X-SERVICE-TOKEN {SERVICE_TOKEN}"},
+            json={"discord_user_id": uid},
+        )
+    res = resp.json()
+
+    if resp.status_code == 400:
+        return {
+            "type": 4,
+            "data": {
+                "flags": EPHEMERAL,
+                "content": f"It doesn't look like your discord account is linked to your league history. Run /link to connect.",
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)